sites/website/src/docs/fast-foundation/getting-started.md

```diff
@@ -36,6 +43,7 @@ By default, the background color and text color won't be set by the `fast-design
 ```
 
 ### Add Components
+
 Add any FAST elements (or any element) as a child of the `fast-design-system-provider`. That's it! For a comprehensive list of all elements, see the Components section.
```

Comment:
Any time referencing another section I find helpful to provide a link to eliminate any hurdle. In this case, hunting for the section.

Reply:
I didn't really change content of other articles, but I can change this. The issue is that there's no "landing page" for the components besides this getting started page. So, we'd have to pick a specific component to link to.

Reply:
Should we have a landing page?  Seems like it might be a good idea to solve a this common problem you described.  I'm a fan of landing pages because they give you another opportunity to dive just al little deeper into web components as a group, pros/cons, strategy, etc that may add value.

Reply:
This page technically is the landing page for fast foundation. We could have another one in the components sub-section but I'm not sure exactly what content to put there. Let me think about that over the weekend.

Reply:
Another idea on this might be a good spot to discuss architecture of how the 4 packages work together along with the strategy, pros/cons I mentioned previously.